	- Ensure correct (HOST-)FLAGS when compiling the intermediates
	- Drop version test, this cant work anyway since there is no aufs3.12 branch anywhere
--- w-aufs-util-3.x-rcN-050614-1.orig/Makefile	2014-05-06 15:36:53.000000000 +0200
+++ w-aufs-util-3.x-rcN-050614-1/Makefile	2014-06-05 14:32:53.496611173 +0200
@@ -49,7 +49,7 @@ LibUtilObj = perror.o proc_mnt.o br.o pl
 LibUtilHdr = au_util.h
 export
 
-all: ver_test ${Man} ${Bin} ${Etc}
+all: ${Man} ${Bin} ${Etc}
 	${MAKE} -C libau $@
 	ln -sf ./libau/libau*.so .
 
@@ -85,6 +85,9 @@ aufs.5: aufs.in.5 c2tmac
 	chmod a-w $@
 
 c2sh c2tmac ver: CC = ${HOSTCC}
+c2sh c2tmac ver: CFLAGS = ${CFLAGS_FOR_BUILD}
+c2sh c2tmac ver: LDFLAGS = ${LDFLAGS_FOR_BUILD}
+c2sh c2tmac ver: CPPFLAGS = ${CPPFLAGS_FOR_BUILD}
 .INTERMEDIATE: c2sh c2tmac ver
 
 Install = install -o root -g root -p
